packages/clerk-js/src/ui/components/OrganizationProfile/RemoveDomainPage.tsx

```diff
@@ -60,7 +66,8 @@ export const RemoveDomainPage = () => {
       successMessage={localizationKeys('organizationProfile.removeDomainPage.successMessage', {
         domain: ref.current?.name,
       })}
-      deleteResource={() => Promise.resolve(domain?.delete())}
+      deleteResource={() => domain?.delete().then((domains as any).unstable__mutate())}
```

Comment:
Does unstable mutate return a function here? Shouldn't it be 
```suggestion
      deleteResource={() => domain?.delete().then(() => (domains as any).unstable__mutate())}
```

Reply:
You are correct, I added the parentheses in the end by mistake. Nice catch